src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPageNew.tsx

```diff
@@ -59,6 +65,65 @@ const FindStatisticsPageNew: NextPage<Props> = ({
     });
   };
 
+  const handleSearchPublications = async (nextSearchTerm: string) => {
+    await router.push(
```

Comment:
Would be a good idea to extract out a function so that we don't repeat ourselves and can ensure we're always correctly calling `router.push`. Something like this might work:

```ts
const updateQueryParams = (query: FindStatisticsPageQuery) => {
    await router.push(
      {
        pathname: '/find-statistics',
        query: query as ParsedUrlQuery,
      },
      undefined,
      {
        scroll: false,
      },
    );
}

// Then we would use it like:

const handleSearchPublication = async (nextSearchTerm: string) => {
  await updateQueryParams({...});
}
```

Note that I've suggested we define a new type `FindStatisticsPageQuery`. This would contain all the query parameters that can be used on this page and we would add this as a type parameter `getServerSideProps` as well:

```ts
export interface FindStatisticsPageQuery {
  ...
}

export const getServerSideProps: GetServerSideProps<Props, FindStatisticsPageQuery> = ({
  query
}) => {
  ...
}
```